peachjam/migrations/0109_add_overturned_and_upheld_predicates.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.2.19 on 2023-11-14 09:13
+
+from django.db import migrations
+
+
+def add_overturned_and_upheld_predicates(apps, schema_editor):
```

Comment:
This migration will fail if this predicate already exists, so it's much better to use `update_or_create`. The goal of the migration is to ensure it exists, and it must tolerate the case where it already does.